SofaScoreAcademy/SofaScoreAcademy/ViewController.swift

```diff
@@ -9,98 +9,78 @@ import UIKit
 import SnapKit
 import SofaAcademic
 
-class LeagueMatchesViewController: UIViewController, BaseViewProtocol {
-    
-    private let stackView = UIStackView()
+class LeagueMatchesViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
jel mozes svaku odvojit u zasebnu npr tu ostavis i onda dodas ispod ekstenzije za ove ostale dvije
```suggestion
class LeagueMatchesViewController: UIViewController {
```